refbox/src/app/mod.rs

```diff
@@ -961,10 +961,16 @@ impl Application for RefBoxApp {
                 } else {
                     let mut tm = self.tm.lock().unwrap();
                     let now = Instant::now();
-                    tm.add_score(color, 0, now);
-                    let snapshot = tm.generate_snapshot(now).unwrap(); // TODO: Remove this unwrap
-                    std::mem::drop(tm);
-                    self.apply_snapshot(snapshot);
```

Comment:
These lines are important ans cannot be skipped. Without them the score displayed will be incorrect until the next change in the time